### PR TITLE
Added GEO section and rule

### PR DESCRIPTION
**Pull request by mrtn78**

### CODE REVIEW

**Review by strijm — CHANGES_REQUESTED** [review]

File: DesignRules.md [hunk 1]
```diff
@@ -679,5 +679,32 @@ Changes in APIs are inevitable. APIs should therefore always be versioned, facil
...
+<div class="rule" id="/core/geo">
+  <p class="rulelab"><b>/core/geo</b>: Use the GEO module for geospatial content</p>
+   <dl>
+      <dt>Statement</dt>
+      <dd>
+         Use the specific GEO module when designing an API for geospatial content and functions.
+      </dd>
+      <dt>Rationale</dt>
+      <dd>
+         <p>The NL API Strategy [[Geospatial Module]] provides rules for publishing geospatial data using Web APIs. Spatial data is data that describes anything with spatial extent (i.e. size, shape or position). Spatial data is also known as location information. [[sdw-bp]] </p>
```
[the remark this answers is not in the document]
Comment (mrtn78) [reply DesignRules.md:695]: ik had deze zin deels overgenomen van https://docs.geostandaarden.nl/api/API-Strategie-mod-geo/#introduction vandaar alleen de focus op publiceren. zal ik het aanpassen of zo laten?

**Review by strijm — APPROVED** [review]

File: DesignRules.md [hunk 2]
```diff
@@ -679,5 +679,30 @@ Changes in APIs are inevitable. APIs should therefore always be versioned, facil
    </dl>
 </div>
 
+## Geospatial
 
+Geospatial content is becomming more common in every modern API. APIs containing [=OGC=] compliant features should therefore be compliant with geospatial specific rules.
```
Comment (lvdbrink) [inline DesignRules.md:684]: typo 'becomming' > should be 'becoming'

File: DesignRules.md [hunk 3]
```diff
@@ -679,5 +679,30 @@ Changes in APIs are inevitable. APIs should therefore always be versioned, facil
...
+      <dt>Implications</dt>
+      <dd>
+         Adherance to this rule needs to be manually verified.
+      </dd>
+      <dt>Rule types</dt>
+      <dd>
+         This is a functional design rule and hence can't be tested automatically.
+      </dd>
+   </dl>
+</div>
```
[the remark this answers is not in the document]
Comment (lvdbrink) [reply DesignRules.md:708]: When reading for a second time, I think this sentence can stay. It informs readers that the geo module is in line with OGC API features which is good info.

**Review by lvdbrink — APPROVED** [review]
I found one typo (becomming > becoming) but other than that it looks good now!

**Pull request merged**